e2e/basic/reverse_proxy_test.go

```diff
@@ -0,0 +1,55 @@
+package basic
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReverseProxy(t *testing.T) {
```

Comment:
For general tests like this I think it is very helpful to have a comment explaining what we are checking for in the test and why that is helpful/valid. 

Looking at this from a place of having not enough context of the reverse proxy, it isn't clear how this test is testing it and why this is an effective test.

Reply:
done